steamcmd/debian/Dockerfile

```diff
@@ -10,22 +10,62 @@ ENV         DEBIAN_FRONTEND=noninteractive
 RUN         dpkg --add-architecture i386 \
             && apt-get update \
             && apt-get upgrade -y \
-            && apt-get install -y tar curl gcc g++ lib32gcc-s1 libgcc-12-dev libgcc-11-dev libcurl4-gnutls-dev:i386 libssl-dev:i386 libcurl4:i386 lib32tinfo6 libtinfo6:i386 lib32z1 lib32stdc++6 libncurses5:i386 libcurl3-gnutls:i386 libsdl2-2.0-0:i386 libsdl2-2.0-0 iproute2 gdb libsdl1.2debian libfontconfig1 telnet net-tools netcat-traditional tzdata numactl xvfb wget tini \
-            && useradd -m -d /home/container container
+            && apt-get install -y \
```

Comment:
Why reformat this like this? It was fine before.

Reply:
This allows you to see duplicate packages more quickly

Reply:
But if it's not wanted, I'll close this PR

Reply:
I would agree that it does make it easier to see i you may have duplicate, but the standard format across the repo is the single line.

I definitely want the locale changes.

Reply:
I don't mean to be a monkey wrench @parkervcp, but if you'll accept my opinion, I believe having the core packages in list form for all the Dockerfiles in the repo is far easier to read and maintain than a single line, especially if there are a lot of packages. It would also make it easier to recognize package changes in future diffs. I agree formatting across the repo should be consistent, but I don't think that should prevent change if it's beneficial.

Reply:
I am not against it. If we do this here then we need to update all the other yolks.